.github/workflows/build-and-push-cliain.yml

```diff
@@ -82,10 +71,12 @@ jobs:
                   (steps.get-ref-properties.outputs.branch-name-flattened == 'main' && 'latest' ||
                   steps.get-ref-properties.outputs.branch-name-flattened) ||
                 matrix.runtime == 'liminal' && steps.get-ref-properties.outputs.sha }}
-        uses: docker/build-push-action@v3.3.0
-        with:
-          context: ./bin/cliain
-          builder: ${{ steps.buildx.outputs.name }}
-          file: ./bin/cliain/Dockerfile
-          push: ${{ github.event_name == 'workflow_dispatch' || github.event_name == 'push' }}
-          tags: ${{ env.REGISTRY }}${{ env.IMAGE_NAME }}:${{ env.TAG }}
+        run: |
+          docker build -t "${{ env.REGISTRY }}${{ env.IMAGE_NAME }}:${{ env.TAG }}" \
+            -f ./bin/cliain/Dockerfile ./bin/cliain
+          if [[ \
+            "${{ github.event_name }}" == "workflow_dispatch" || \
+            "${{ github.event_name }}" == "push" \
+          ]]; then
+             docker push "${{ env.REGISTRY }}${{ env.IMAGE_NAME }}:${{ env.TAG }}"
```

Comment:
This fails since you need to remove `>` in line 69, ie it should be 
```
  TAG: ${{ matrix.runtime == 'standard' &&
                  (steps.get-ref-properties.outputs.branch-name-flattened == 'main' && 'latest' ||
                  steps.get-ref-properties.outputs.branch-name-flattened) ||
                matrix.runtime == 'liminal' && steps.get-ref-properties.outputs.sha }}
``` 
as `>` adds newline

Reply:
Changed.